index.html

```diff
@@ -596,14 +781,35 @@ <h2>Contact</h2>
         }, 250)
       );
 
-      canvas.addEventListener("mousemove", draw);
+      let timeoutReference;
+
+      canvas.addEventListener("mousemove", (e) => {
+        eventRef = e;
+        if (timeoutReference) return;
+
+        timeoutReference = setInterval(() => {
+          draw(e);
+        }, 1);
+      });
       canvas.addEventListener("touchmove", (e) => {
-        e.preventDefault();
-        draw(e);
+        eventRef = e;
+        if (timeoutReference) return;
+
+        timeoutReference = setInterval(() => {
+          draw(e);
+        }, 1);
       });
 
-      canvas.addEventListener("mouseout", () => (isFirstTouch = true));
-      canvas.addEventListener("touchend", () => (isFirstTouch = true));
+      window.addEventListener("blur", () => {
```

Comment:
clearing the draw function only when `blurred` (unfocused) instead of when mouseout happens